src/DependencyInjection/Configuration.php

```diff
@@ -127,6 +127,9 @@ public function getConfigTreeBuilder()
                         ->booleanNode('use_stickyforms')->defaultTrue()->end()
                         ->integerNode('pager_links')->defaultNull()->end()
                         ->scalarNode('form_type')->defaultValue('standard')->end()
+                        ->scalarNode('default_group')->defaultValue('default')->end()
```

Comment:
Please add `info` calls for every node you added.

Reply:
added